src/themes/dark/theme.css

```diff
@@ -421,8 +421,7 @@ html {
     color: #4285f4;
 }
 
-.card:focus .cardBox.visualCardBox,
-.card:focus .cardBox:not(.visualCardBox) .cardScalable {
+.card:focus .cardBox.visualCardBox {
```

Comment:
You broke focus for "slow" devices.
`visualCardBox` and non-`visualCardBox` is "Poster/Thumb card" and "Poster/Thumb"
`visualCardBox` tells to make border over whole card (with title).

For testing set some TV user agent.
`Mozilla/5.0 (SMART-TV; Linux; Tizen 4.0) AppleWebKit/537.36 (KHTML, like Gecko) SamsungBrowser/2.1 Chrome/56.0.2924.0 TV Safari/537.36`

Reply:
Oh, didn't mean to remove that :/